tracer/test/test-applications/integrations/Samples.HotChocolate/Samples.HotChocolate.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
afaik, the target frameworks are set by the build.props at test-applications level so we usually dont set them in the sample csproj